src/plugins/intel_cpu/tests/functional/shared_tests_instances/low_precision_transformations/fake_quantize_with_dq_not_optimal_transformation.cpp

```diff
@@ -42,7 +42,7 @@ const std::vector<FakeQuantizeWithNotOptimalTransformationTestValues> fakeQuanti
         {},
         "FP32"
     },
-    {
+/*  {
```

Comment:
Commented code detected. Please either uncomment the code frame, or make a ticket and disable the test via the common mechanism (e.g. https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp#L178, though it is commented the mechanics is the same).